src/overrides.jl

```diff
@@ -171,3 +171,33 @@ function Graphs.induced_subgraph(
     newg.weights = new_weights
     return newg, Vector{E}(vlist)
 end
+
+function induced_subgraph(g::T, elist::AbstractVector{U}) where T <: AbstractSimpleWeightedGraph where U <: AbstractEdge
```

Comment:
```suggestion
function Graphs.induced_subgraph(g::T, elist::AbstractVector{U}) where T <: AbstractSimpleWeightedGraph where U <: AbstractEdge
```
(Otherwise it won't extend the function in Graphs.jl but just define a new, distinct one since it hasn't been explicitly imported. Unrelated to this PR, but something similar seems to be afflicting [this line](https://github.com/JuliaGraphs/SimpleWeightedGraphs.jl/blob/c508a11e8d10ec7421354336c8004b1204f15dc6/src/overrides.jl#L6) of `src/overrides.jl`)

Reply:
Taking care of it, thanks!

Reply:
And surprisingly, `degree_matrix` is not part of Graphs.jl!

Reply:
https://github.com/search?q=repo%3AJuliaGraphs%2FGraphs.jl+degree_matrix&type=code